docs/spec_draft.md

```diff
@@ -1443,6 +1445,169 @@ specification. Numeric precision is implementation-defined.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.scatter
+
+### Semantics
+
+Generate `results` which is the values of the `inputs` operand, with several
+slices at indices specified by `scatter_indices`, updated with the values in
+`updates` using `update_computation`.
+
+The following diagram shows how elements in `updates[k]` map on elements in
+`inputs[k]` using a concrete example. The diagram picks a few example
+`updated[k[` indices and explains in detail which `inputs[k]` indices they
+correspond to.
+
+<img align="center" src="spec_images/scatter.png" />
+
+More formally, for any `k`, `updates[k][update_index] =
+inputs[k][input_index]` where:
```

Comment:
"for any `k`, `updates[k][update_index] = inputs[k][input_index]`" seems out of place. 

First, `updates` are never on the left-hand side of an assignment. Secondly, the scheme by which updates are happening is more complicated than just assignment, so let's not get into that in this paragraph. Proposed reformulation:

```
More formally, for all `update_index` from the index space of `updates[0]`,
`result_index` is computed as follows:
```

Also, let's use `updates[0]` and `results[0]` consistently below, because all `updates` have the same shapes and all `results` have the same shapes.